alphadia/constants/default.yaml

```diff
@@ -141,8 +141,8 @@ optimization_manager:
 
 # This section controls transfer learning
 # currently only the library is created with transfer learning
-transfer_learning:
-  # if true, the library is created with transfer learning
+transfer_library:
```

Comment:
(very) strictly speaking, this a a breaking change

![image](https://github.com/MannLabs/alphadia/assets/82171591/f81213c5-1ad1-4da5-bfdb-36318bca8cde)

Reply:
I could settle for 1.7.0?

Reply:
fair enough, I guess there will be a lot of these "strictly speaking breaking changes" in the near future, and we don't want to blow up the major version too much
(writing this using Firefox 126.0.1 btw :-D )